app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt

```diff
@@ -379,43 +374,10 @@ fun PlayerScreen(
             }
         }
     }
-    App.context?.let { recently_played_jsoncache(it,"recently_played",currentlyPlayingSong) }
-    App.context?.let { getFile(it,"recently_played") }
-}
-fun recently_played_jsoncache(context: Context, key: String, value: Song) {
-    val cacheDir: File = context.cacheDir
-    val gson = Gson()
-    val json = gson.toJson(value)
-    val file=File(cacheDir,key)
-    try {
-        val fileWriter= FileWriter(file,true)
-        fileWriter.write("$json,")
-        fileWriter.close()
-    }catch (e:IOException)
-    {
-        println(e)
+    var cache= App.context?.let { CacheService<Song>(it, "recentlyPlayed") }
```

Comment:
Could you please put the key name in the Constants file and not hardcode it?

Reply:
Yeah sure @akshaaatt i will do it

Reply:
@rishuriya also, please join the `#metabrainz` channel on IRC. I read your messages on personal but could not reply to them because your nick didn't exist anymore. Please go through https://docs.google.com/document/d/1hY5oloIiANeXg1R9oSBIr2ZUHSlm7LU8qy6tW1VJMQg/edit?usp=sharing for anymore questions

Reply:
@akshaaatt added the key in the constant. Can you please review it again, and if any other change required, else please merge it.

Reply:
Ok sure @akshaaatt , i already joined it but i am not able to view any msg.